pymatgen/io/qchem/tests/test_inputs.py

```diff
@@ -45,6 +45,51 @@ def test_molecule_template(self):
 
         assert molecule_actual == molecule_test
 
+    def test_multi_molecule_template(self):
+        self.maxDiff = None
+        species = ["C", "C", "H", "H", "H", "H"]
+        coords_1 = [
+            [0.000000, 0.000000, 0.000000],
+            [1.332000, 0.000000, 0.000000],
+            [-0.574301, 0.000000, -0.928785],
+            [-0.574301, 0.000000, 0.928785],
+            [1.906301, 0.000000, 0.928785],
+            [1.906301, 0.000000, -0.928785],
+        ]
+        coords_2 = [
+            [0.000000, 4.000000, 0.000000],
+            [1.332000, 4.000000, 0.000000],
+            [-0.574301, 4.000000, -0.928785],
+            [-0.574301, 4.000000, 0.928785],
+            [1.906301, 4.000000, 0.928785],
+            [1.906301, 4.000000, -0.928785],
+        ]
+
+        mol_1 = Molecule(species, coords_1, charge=1)
+        mol_2 = Molecule(species, coords_2)
+        molecule_test = QCInput.molecule_template([mol_1, mol_2])
+        molecule_actual = """$molecule
+ 1 2
+--
+ 1 2
+ C      0.0000000000      0.0000000000      0.0000000000
+ C      1.3320000000      0.0000000000      0.0000000000
+ H     -0.5743010000      0.0000000000     -0.9287850000
+ H     -0.5743010000      0.0000000000      0.9287850000
+ H      1.9063010000      0.0000000000      0.9287850000
+ H      1.9063010000      0.0000000000     -0.9287850000
+--
+ 0 1
+ C      0.0000000000      4.0000000000      0.0000000000
+ C      1.3320000000      4.0000000000      0.0000000000
+ H     -0.5743010000      4.0000000000     -0.9287850000
+ H     -0.5743010000      4.0000000000      0.9287850000
+ H      1.9063010000      4.0000000000      0.9287850000
+ H      1.9063010000      4.0000000000     -0.9287850000
+$end"""
+
+        self.assertEqual(molecule_test, molecule_actual)
```

Comment:
Since we use `pytest` to run tests, could simplify to

```py
assert molecule_test == molecule_actual
```

Reply:
Sure. Small change, happy to do it.